Cargo.toml

```diff
@@ -142,6 +142,7 @@ debug-assertions = false
 #
 #     [patch."https://github.com/servo/<repository>"]
 #     <crate> = { path = "/path/to/local/checkout" }
+mozangle = { git = "https://github.com/atbrakhi/mozangle.git", branch = "shared_object" }
```

Comment:
why you don't use `https://github.com/servo/mozangle` ?

Reply:
because this PR is meant for testing PR to mozangle: https://github.com/servo/mozangle/pull/71 and while this PR is not landed, for testing purposes we need to use source repo of PR (which is atbrakhi's personal fork of mozangle repo).